charts/rancher-turtles/templates/rke2-bootstrap.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if and (index .Values "cluster-api-operator" "cluster-api" "enabled") (index .Values "cluster-api-operator" "cluster-api" "rke2" "enabled") }}
```

Comment:
This was mainly based of off the core-yaml file and that has changed in https://github.com/rancher/turtles/pull/458/files, so this and `rke2-controlplane.yaml` files also needs the same changes wrt namespace check and hook